core/src/main/scala/kafka/server/ClientRequestQuotaManager.scala

```diff
@@ -46,7 +49,8 @@ class ClientRequestQuotaManager(private val config: ClientQuotaManagerConfig,
   private val exemptMetricName = metrics.metricName("exempt-request-time",
     QuotaType.Request.toString, "Tracking exempt-request-time utilization percentage")
 
-  lazy val exemptSensor: Sensor = getOrCreateSensor(ClientRequestQuotaManager.ExemptSensorName, exemptMetricName)
+  val exemptSensor: Sensor = getOrCreateSensor(ClientRequestQuotaManager.ExemptSensorName,
```

Comment:
Since there should be only one global exemptSensor, which doesn't need to expire.
Maybe it makes more sense to move this field into the singleton object ClientRequestQuotaManager, have it inherit from the KafkaMetricsGroup and create the exemptRequestRate via the newMeter api, similar to

https://github.com/linkedin/kafka/blob/09a265dc4d3460b611bfe44f3a038e25bb66a707/core/src/main/scala/kafka/controller/KafkaController.scala#L3157

Reply:
exemptSensor is associated with Metrics metrics which is a parameter for the class constructor, so it is not easy to move to object. ClientRequestQuotaManager is essentially only one instance in KafkaServer. I do not feel it is necessary in this PR to change from using Sensor to KafkaMetricsGroup.